org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/editor/quickfix/QuickFixTestHelper.xtend

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.xtext.ide.tests.editor.quickfix
+
+import com.google.common.base.Preconditions
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.resource.ResourceSet
+import org.eclipse.xtext.ide.tests.serializer.ChangeSerializerTestHelper
+
+class QuickFixTestHelper extends ChangeSerializerTestHelper {
```

Comment:
Xtext is trying to reduce the amount of xtend code in the codebase. This class doesn't gain too much from being xtend, could you write it in Java instead?

Reply:
will do